README.md

```diff
@@ -222,6 +222,11 @@ Also: you might be able to do real damage with the CLI to all your data,
 because you have the full power of python at your fingertips.
 Enjoy!
 
+### Shortcuts
+Shortcuts for each action are indicated in square brackets in the app(for example `[F]` to set the file name).
+Shortcuts are inactivated when you input a text or a number. Press enter to end input and activate shortcuts again.
+Press `Ctrl+Q` to close the app.
+
```

Comment:
Would be good to tell the user how they can find out on what shortcuts are set. They are implemented in the tooltips, no?

Reply:
Not in the tooltips, in the labels. For example 'File name [F]' or 'Capture image [Space]'

Reply:
ah ok, that resolves this questions